packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -179,9 +217,33 @@ contract SortedOracles is ISortedOracles, Ownable, Initializable {
       address(0)
     );
     emit OracleReported(token, msg.sender, now, value, DENOMINATOR);
+    recomputeRate(token);
+  }
+
+  function recomputeRate(address token) internal {
+    uint256 originalMedian = limitedMedianRate[token];
     uint256 newMedian = rates[token].getMedianValue();
+    if (limitedMedianRateTimestamp[token] != 0 && maxMedianChangeRatePerDay.unwrap() != 0) {
+      uint256 td = now.sub(limitedMedianRateTimestamp[token]);
+      if (td > 1 days) td = 1 days;
+      FixidityLib.Fraction memory timeFraction = FixidityLib.newFixedFraction(td, 1 days);
+      FixidityLib.Fraction memory maxChangeRate = timeFraction.multiply(maxMedianChangeRatePerDay);
```

Comment:
Is this exactly what we want? I would've thought `maxMedianChangeRatePerDay` would compound

e.g. if `maxMedianChangeRatePerDay` was 10%, and the median rate was 1 at t=0, at t=5 it could be set to 1 * 1.1^5 or 1 / 1.1^5

Reply:
Yeah, it won't compound now, so if you update more frequently, you get more change.
Perhaps we have to implement exponent function?

Reply:
Looks like there is already a precompile for that

Reply:
I think there is a precompile that could be useful, going to test that.